common/src/main/java/com/wynntils/screens/skillpointloadouts/SkillPointLoadoutsScreen.java

```diff
@@ -0,0 +1,538 @@
+                    resetSaveButtons();
+                },
+                this,
+                saveNameInput);
+        this.addRenderableWidget(saveNameInput);
+
+        saveAssignedButton = new SaveButton(
+                (int) (dividedWidth * 49),
+                (int) (dividedHeight * 24),
+                (int) ((dividedWidth * 53) - (dividedWidth * 49)),
+                BUTTON_HEIGHT,
+                Component.translatable("screens.wynntils.skillPointLoadouts.save"),
+                this,
+                Models.SkillPoint::saveCurrentSkillPoints);
+        this.addRenderableWidget(saveAssignedButton);
+
+        saveBuildButton = new SaveButton(
+                (int) (dividedWidth * 54),
+                (int) (dividedHeight * 24),
+                (int) ((dividedWidth * 59) - (dividedWidth * 54)),
+                BUTTON_HEIGHT,
+                Component.translatable("screens.wynntils.skillPointLoadouts.saveBuild"),
+                this,
+                Models.SkillPoint::saveCurrentBuild);
+        this.addRenderableWidget(saveBuildButton);
+
+        loadButton =
+                new WynntilsButton(
+                        (int) (dividedWidth * 35),
+                        (int) (dividedHeight * 52),
+                        (int) ((dividedWidth * 44) - (dividedWidth * 35)),
+                        BUTTON_HEIGHT,
+                        Component.translatable("screens.wynntils.skillPointLoadouts.load")) {
+                    @Override
+                    public void onPress() {
+                        Models.SkillPoint.loadLoadout(selectedLoadout.key());
+                    }
+                };
+        this.addRenderableWidget(loadButton);
+
+        deleteButton =
+                new WynntilsButton(
+                        (int) (dividedWidth * 45),
+                        (int) (dividedHeight * 52),
+                        (int) ((dividedWidth * 51) - (dividedWidth * 45)),
+                        BUTTON_HEIGHT,
+                        Component.translatable("screens.wynntils.skillPointLoadouts.delete")
+                                .withStyle(ChatFormatting.RED)) {
+                    @Override
+                    public void onPress() {
+                        Models.SkillPoint.deleteLoadout(selectedLoadout.key());
+                        setSelectedLoadout(null);
+                        populateLoadouts();
+                    }
+                };
+        this.addRenderableWidget(deleteButton);
+
+        convertButton =
+                new WynntilsButton(
+                        (int) (dividedWidth * 52),
+                        (int) (dividedHeight * 52),
+                        (int) ((dividedWidth * 59) - (dividedWidth * 52)),
+                        BUTTON_HEIGHT,
+                        Component.translatable("screens.wynntils.skillPointLoadouts.convert")) {
+                    @Override
+                    public void onPress() {
```

Comment:
I think I previously commented on how these buttons should be separate classes, not anonymous ones. I think you only converted the first one, and forgot the others.